third_party/googlelog/BUILD

```diff
@@ -5,4 +5,5 @@ licenses(["notice"])  # BSD
 cc_library(
     name = "config_h",
     hdrs = ["src/config.h"],
+    includes = ["src"],
```

Comment:
This is incorrect.  `includes` adds the include transitively to all targets which depend on this target, but the config.h file is an implementation detail which should *not* be added or included by dependents of this target.

Reply:
I used `strip_include_prefix` instead.